templates/waf_web_acl.cfn.yml

```diff
@@ -42,13 +42,13 @@ Resources:
             RateBasedStatement:
               AggregateKeyType: "IP"
               EvaluationWindowSec: 60
-              Limit: 60
+              Limit: 600
               ScopeDownStatement:
                 RegexMatchStatement:
                   FieldToMatch:
                     UriPath: {}
                   RegexString:
-                    /api/
+                    /api/|/user/login
```

Comment:
``\/api\/[3\/]*action\/am_following_user|\/user``
or
``/api/[3/]*action/am_following_user|/user``

since he only poked at the follow user api endpoint, not all.

Reply:
Done, although I'm limiting it to the login endpoint, not everything under `/user`